install.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/sh
+
+function install_fail() {
+    echo "Installation failed"
+    exit 1
+}
+
+
+function Help(){
+    echo "Usage: install.sh [option]"
+    echo "Options:"
+    echo "  -h: Show this help message and exit"
+    echo "  -d: Install only dependencies"
+    echo "  -p: Install only python dependencies (including playwright)"
+}
+
+
+
+while getopts ":hydp" option; do
+    case $option in
+        h)
+            Help
+            exit 0;;
+        y)
+            ASSUME_YES=1;;
+        d)
+            DEPS_ONLY=1;;
+        p)
+            PYTHON_ONLY=1;;
+        \?)
+            echo "Invalid option: -$OPTARG" >&2
+            Help
+            exit 1;;
+        :)
+            echo "Option -$OPTARG requires an argument." >&2
+            Help
+            exit 1;;
+    esac
+done
+
+
+
+function install_macos(){
+    if [ ! command -v brew &> /dev/null ]; then
+        NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"
```

Comment:
you want to install brew here, not uninstall
```suggestion
        NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```